list_people.rb

```diff
@@ -1,7 +1,11 @@
 class ListPeople
   def list_people(people)
     people.each do |p|
-      puts "[#{p.class}] Name: #{p.name}, ID: #{p.id}, Age: #{p.age}"
+      if p['specialization']
+        puts "[teacher] Name: #{p['name']}, ID: #{p['id']}, Age: #{p['age']}"
+      else
+        puts "[student] Name: #{p['name']}, ID: #{p['id']}, Age: #{p['age']}"
+      end
```

Comment:
- [ ] Inside your `json` file I suggest that you add `json.class` like if is `people.json` file you can outputs:
`{"json_class":"Student","name":"Sam","age":"14","parent_permission":true}`
`{"json_class":"Teacher","name":"Fox","age":"56","parent_permission":true,"specialization":"Starships"}`